go.mod

```diff
@@ -4,7 +4,9 @@ go 1.14
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.5.7
+	github.com/ava-labs/avalanche-testing v0.9.6
 	github.com/ava-labs/avalanchego v1.0.4-update-id-2
+	github.com/ava-labs/gecko v0.6.1-rc.1
```

Comment:
this should be avalanchego 1.0.5+ ?

Reply:
Yes, good catch. Idk why this gets overrided sometimes when building.